C-Plus-Plus/heaps/Median_in_a_running_stream_of_integers.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+Median of a stream of running integers
+======================================
+Print the median of given numbers while keep taking in new numbers
+
+Concept:
+We maintain two heaps - a minHeap and a maxHeap. minHeap contains numbers greater than the current median and
+maxHeap contains numbers less than equal to the current median.
+Note that if the number of elements in both the heaps are equal then median will be equal to the average of the
+topmost elements of the two heaps. If any heap contains more elements than the other heap then its topmost element will
+be the current median. Also note that the maximum difference in the number of elements between the two heaps should always be
+less than equal to 1.
+
+|n(minHeap) - n(maxHeap)|<=1
+
+If we are in a situation where inserting an element causes increase in this difference, then we must transfer an element
+from one heap to other to settle this.
+
+Time Complexity: O(nlogn)
+Space Complexity: O(n)
+*/
+
+#include <iostream>
+#include<queue>
+using namespace std;
+
+int main() {
```

Comment:
Shift your entire logic to a different function. And use main() just for the Driver Code.